README.md

```diff
@@ -80,6 +80,10 @@ AWS_REGION: us-east-1
 SSL_CERT: /tmp/ssl-cert.pem
 ```
 
+**Note:** Since the buildpack injects secrets into the application runtime environment using the [bash export method](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-export),
+environment variable names included in the secret.yml file **must** be valid shell variable names. In particular,
+they may contain upper or lowercase letters, numbers, and underscores **only**.
```

Comment:
This section (and probably the others here) need some reasonable line feeds / wraps added

Reply:
yeah, I modeled it after the surrounding sections. I can take a stab at cleaning the file up and add a commit for that.

Reply:
@sgnn7 fixed up.